test/e2e/specs/updates/scaleoutin/scaleoutin.go

```diff
@@ -0,0 +1,216 @@
+package scaleoutin
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	apiappsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/fakerp"
+	"github.com/openshift/openshift-azure/pkg/util/randomstring"
+	"github.com/openshift/openshift-azure/pkg/util/ready"
+	"github.com/openshift/openshift-azure/test/clients/azure"
+	"github.com/openshift/openshift-azure/test/clients/openshift"
+	"github.com/openshift/openshift-azure/test/e2e/specs/updates"
+)
+
+var _ = Describe("Scale Out/In E2E tests [ScaleOutIn] [Update]", func() {
```

Comment:
this should be tagged to run on every PR and against the real RP

Reply:
done, tags are: `[ScaleUpDown] [Real] [LongRunning] [EveryPR]`